### ISSUE TITLE
Support for sending/receiving MIDI clock?

### ISSUE THREAD

**Issue by Hans0lo** [pschatzmann/arduino-midi#3]
Hi Phil,
first: thnx for your work!

I wonder, as I searched for the word 'clock' here:
https://pschatzmann.github.io/arduino-midi/html/annotated.html

but it was not found, so I assume that this library does not support sending/receiving clock?

**Comment by pschatzmann** [comment]
No, so far I did not have any need to support any System messages.
However it should not be too difficult to add...

**Comment by Hans0lo** [comment]
thnx for the speedy reply - can you eventually give me some hints on how/where in the package system messages should be added ?
(i do know MIDI specs quite well, but am not too experienced in c++ coding.... so i feel a bit overwhelmed when i try to find the right place in all of the modules that are part of this lib)

**Comment by pschatzmann** [comment]
You would extend MidiCommon with the system messages e.g.
- systemTimingClock()
- systemStart()
- systemEnd()
- etc
You would extend the MidiParser class to handle
- onSystemTimingClock()
- onSystemStart()
- onSystemEnd()
-etc

**Comment by Hans0lo** [comment]
thnx a lot Phil!
